### ISSUE TITLE
Illustration of a MiniApp page

### ISSUE THREAD

**Issue by xfq** [w3c/miniapp#190]
Because the concept of a [page](https://w3c.github.io/miniapp-packaging/#dfn-page) in a MiniApp is not exactly the same as that of the web, it would be more intuitive to have a picture to indicate what a page in a MiniApp is.

**Comment by espinr** [comment]
Good point @xfq , 
I created this picture in the past. 
![MiniApp components and page structure](https://raw.githack.com/espinr/miniapp-packaging/image-page-components/images/miniapp-pages-components.svg)

Please MiniApp vendors, have a look and suggest changes if needed. Based on this picture, we also need to clarify the concept of _component_. Or just using another term?

**Comment by xfq** [comment]
Is it possible that multiple pages share the same CSS (such as the general theme for the MiniApp) or share the same component (such as a list component)?

**Comment by espinr** [comment]
> Is it possible that multiple pages share the same CSS (such as the general theme for the MiniApp) or share the same component (such as a list component)?

Yes, I've tried to represent it on the same diagram with also some examples of shared resources (image and stylesheets). What about this?

![MiniApp components representation](https://raw.githack.com/espinr/miniapp-packaging/image-page-components/images/miniapp-pages-components.svg)

Suggestions for improving the diagram are welcome. Thanks!

**Comment by xfq** [comment]
LGTM. Thank you!
